### ISSUE TITLE
Delay before show and before hide

### ISSUE THREAD

**Issue by slavafomin** [720kb/angular-tooltips#153]
Hey!

It would be really cool to allow developers to specify delay before tooltip is shown and/or hidden.

**Comment by espeandr** [comment]
+1

**Comment by 45kb** [comment]
Yes this could be very useful, at the moment you can set a delay for when is shown, you can work around the CSS 
This is the CSS animation involved : https://github.com/720kb/angular-tooltips/blob/master/lib/angular-tooltips.scss#L13

Afaik there is an `animation-delay` CSS rule that could help for the moment.

Then could be great to have new attributes `tooltip-show-delay` & `tooltip-hide-delay` to just set milliseconds you would.

Feel free to PR if you try something 👍

**Comment by 45kb** [comment]
https://github.com/720kb/angular-tooltips/issues/158
